src/loops/simple_programs_with_repitition/easy/q6/CountItUpTest.java

```diff
@@ -0,0 +1,38 @@
+package loops.simple_programs_with_repitition.easy.q6;
+
+import global.variables.*;
+import global.variables.clauses.*;
+import global.BaseTest;
+
+public class CountItUpTest extends BaseTest {
+    
+    String input1 = "This is just for testing. Do not copy!";
+    String input2 = "s";
+
```

Comment:
I would pick a different letter than s, or put a capital s in the test string. As it is, if they misplace the .toLowerCase() it would still pass the tests.

Reply:
this should be resolved when inputTest is implemented